hazelcast/vector.py

```diff
@@ -15,6 +15,16 @@ class Type(enum.IntEnum):
 
 @dataclasses.dataclass
 class Vector:
+    """Vector represents a named vector of floats.
+
+    Args:
+        name: Name of the vector.
+            To use the default vector index, name must be set
+            to the blank string (``""``)
+        type: Type of the vector.
+            Currently only ``Type.DENSE`` is supported.
```

Comment:
Do we have a vector types description somewhere? I was unable to find it in the java docs either

Reply:
`DENSE` is just a regular vector. `SPARSE` vectors would be an optimization which are not supported yet.

Reply:
@yuce Do we have python documentation for Type enum(I guess it's a Enum) members? We should have imo

Reply:
Added some docs at: https://github.com/hazelcast/hazelcast-python-client/pull/678/commits/75ff67f5ca7811e094e2a3603b04e59f51eeceab